test/common/modules/live_objects_helper.js

```
@@ -0,0 +1,172 @@
+'use strict';
```

Biome

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)